x-pack/plugins/enterprise_search/server/lib/ml/fetch_ml_models.ts

```diff
@@ -207,7 +207,7 @@ const getUserFriendlyTitle = (modelId: string, modelType: string) => {
   return MODEL_TITLES_BY_TYPE[modelType] !== undefined
     ? MODEL_TITLES_BY_TYPE[modelType]!
     : modelId === LANG_IDENT_MODEL_ID
-    ? 'Lanugage Identification'
+    ? 'Language Identification'
```

Comment:
I forgot to add i18n to this part of the code, please use this:
```suggestion
    ? i18n.translate('xpack.enterpriseSearch.content.ml_inference.lang_ident', {
    defaultMessage: 'Language Identification',
  })
```